README.md

```diff
@@ -47,245 +17,138 @@ After you're finished please remove all the comments and instructions!
     - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
   - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
+- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
+<!-- DESCRIPTION -->
+## 📖 About the Project <a name="#about-project"></a>
 
-# 📖 [your_project_name] <a name="about-project"></a>
+This project will use a relational database to create the data structure for a vet clinic. I'll start with a single table and at the end, I'll create a complete database with data about:
 
-> Describe your project in 1 or 2 sentences.
+    animals
+    animals' owners
+    clinic employees
+    visits
 
-**[your_project__name]** is a...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
 <details>
-  <summary>Server</summary>
+  <summary>PostgreSQL</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://www.postgresql.org">PostgreSQL</a></li>
   </ul>
 </details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+<!-- FEATURES -->
 
-<!-- Features -->
+### ⭐️ Key Features <a name="key-features"></a>
 
-### Key Features <a name="key-features"></a>
+    Create animals table.
+    Query and update animals table.
+    Query multiple tables.
+    Add "join table" for visits.
+    Database performance audit.
+    Add database schema diagram.    
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
```

Comment:
[OPTIONAL] Since this project will not be deployed, I think it would be best if your removed the entire live demo section. It will not be needed in this milestone therefore it is quite unnecessary.

Reply:
Copied, sir **@KDlamini** 

Thanks for the feedback